src/ui-kit/form-controls/sam-sds-autocomplete/autocomplete-search/autocomplete-search.component.ts

```diff
@@ -23,17 +23,13 @@ export class SAMSDSAutocompleteSearchComponent implements ControlValueAccessor {
   /**
    * Ul list of elements 
    */
-  @ViewChild('resultsList', {static: true}) resultsListElement: ElementRef;
+  @ViewChild('resultsList', {static: false}) resultsListElement: ElementRef;
```

Comment:
This was the cause for undefined error in console as user tries to query for more items. Setting static to true indicates that we want to query for this element before change detection runs. However, this element is displayed conditionally in the template based on results, so we rely on change detection to display the element, and should query for it post change detection.